app/src/components/login/Login.tsx

```diff
@@ -1,10 +1,95 @@
-import React from "react";
+import React, { useState } from "react";
+import Input from "../input";
+import isEmailValid from "../../utils/isEmailValid";
+import { isPasswordValid } from "../../utils/passwordValidations";
 import "./Login.css";
+import Button from "../button";
+import { Link } from "react-router-dom";
+import googleIcon from "../../images/googleicon.svg";
+import axios from "axios";
+import getInvalidPasswordMessage from "../../utils/getInvalidPasswordMessage";
+import getInvalidEmailMessage from "../../utils/getInvalidEmailMessage";
 
 function Login() {
+  const [email, setEmail] = useState("");
+  const [password, setPassword] = useState("");
+  const [passwordErrorMessage, setPasswordErrorMessage] = useState("");
+  const handleSignin = async () => {
+    const errors = !isEmailValid(email) || isPasswordValid(password);
+    if (!errors) {
+      let req;
+      let reqError;
+      try {
+        req = await axios({
+          method: "post",
+          url: "/auth/local",
+          data: {
+            email,
```

Comment:
It might be a good idea to `.toLowerCase()` emails sent in the request?

Reply:
Maybe better to do it in the back-end? seeing as it's bad practise to trust input from the client anyway.